ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -528,7 +528,7 @@ tests:
 - as: e2e-agent-sno
   cluster: build05
   optional: true
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg/.*|terraform/providers)/(alibabacloud|aws|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  run_if_changed: /agent/
```

Comment:
If we want to trigger the job only for specific agent code changes, shouldn't be `.*/agent/` (I'm not sure if the regex pattern must be escaped, from the prow [test](https://github.com/kubernetes/test-infra/blob/fe4f3e2158449e8a1b28834bbd2836102d30683f/prow/config/jobs_test.go#L199) seems so, in such case should be then `.*\/agent\/`)?

Reply:
Golang regexp doesn't have separate match/search functions, so `MatchString()` [matches anywhere](https://pkg.go.dev/regexp@go1.19.3#Regexp.Match) in the string. That's why the previous expression had `^` to match only at the start.
`/` has no special meaning in regexps, so it shouldn't be escaped.

Reply:
Ok thanks